src/gl/metal/vcRenderShaders.cpp

```diff
@@ -3,8 +3,8 @@
 const char *const g_VisualizationFragmentShader = "visualizationFragmentShader";
 const char *const g_VisualizationVertexShader = "visualizationVertexShader";
 
-const char *const g_FXAAFragmentShader = "fxaaFragmentShader";
-const char *const g_FXAAVertexShader = "fxaaVertexShader";
+const char *const g_PostEffectsFragmentShader = "postEffectsFragmentShader";
+const char *const g_PostEffectsVertexShader = "postEffectsVertexShader";
```

Comment:
Don't you need to modify the `.metal` file too?

Reply:
Nope.

With the expectation that we'll get shader cross-compilation working soon, getting these shaders working for metal is a bit redundant.